src/main/webapp/js/issues-detail.js

```diff
@@ -165,12 +164,14 @@
     var activeTab = localStorage.getItem('activeTab');
     if (activeTab) {
         detailsTabs.find('a[href="' + activeTab + '"]').tab('show');
+    } else {
+        detailsTabs.find('li:first-child a').tab('show');
     }
 
     /**
      * Stores the order of every table in the local storage of the browser.
      */
-    var tables = $('#statistics').find('table').not('#details');
+    var tables = $('#statistics').find('table').not('.details');
```

Comment:
Or even better `#list` since the ordering for the details should be preserved, just not for the list.

Reply:
Will be changed to .ignore-column-sorting, too. See above.